tests/ten_minutes_wakt_test.js

```diff
@@ -0,0 +1,13 @@
+import test from 'ava';
+import isValidWalk from '../src/ten_minutes_walk.js';
+
+test('Walk should be valid', t => {
+  t.is(isValidWalk(['n','s','n','s','n','s','n','s','n','s']), true);
+  t.is(isValidWalk(['w','s','e','n','e','w','w','s','e','n']), true);
+});
+
+test('Walk not valid', t => {
```

Comment:
Could be split into:
1. Person ends up in other coordinates than 0,0 after 10 steps
2. Walk is not 10 minutes
3. Walk contains invalid directions